docker/Dockerfile

```diff
@@ -9,6 +9,7 @@ FROM confluentinc/cp-kafka-connect-base:7.4.0
 
 COPY --from=builder /tmp/jmx_exporter /opt/
 COPY confluent-connect-libs/ /usr/share/java/confluent-connect/
+COPY backup-cli-libs/ /usr/share/java/backup-cli/
```

Comment:
I would not move it to the `/usr/share/java/` dir since this is used by kafka-connect. I would rather move it to `/opt/`